src/kata6.joinNames.js

```diff
@@ -1,3 +1,14 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+	const nameArray = namesObj.map(function (x) {
+		return x.name;
+	});
```

Comment:
You could do with updating the syntax here to use the arrow function format. Also, its always good to use proper readable variable names, even if it is only in a temporary fashion like this:
```
  const nameArray = namesObj.map((person) => person.name);
```
Just makes it more readable and intuitive